message_ix/cli.py

```diff
@@ -0,0 +1,13 @@
+import argparse
+
+from message_ix.config import config
+
+
+def config():
```

Comment:
Would it make sense to also copy the tutorial folder to the new location, in addition to the `model` folder? And pack of folders within a `message_ix` folder?